eng/versioning/version_client.txt

```diff
@@ -21,6 +21,7 @@ com.azure:azure-core-test;1.3.1;1.4.0-beta.1
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.5;1.0.0-beta.6
 com.azure:azure-cosmos;4.1.0;4.2.0-beta.1
 com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
+com.azure:azure-cosmos-encryption;0.9.0-beta.1;0.9.0-beta.1
```

Comment:
nit: This is new library, let's set the version to 1.0.0-beta.1

Reply:
@JimSuplizio we are not indenting to ship this for the time being (as this is under development for now). long term once ready will be shipped. 

What's the good starting version for such scenario? 1.0.0-beta.1?

Reply:
moved to  1.0.0-beta.1 thanks @JimSuplizio